test/test-application/src/index.ts

```diff
@@ -87,6 +96,58 @@ async function testShowcase() {
   await testWait(restClient);
 }
 
+function getStreamingSequenceRequest(){
+  const request = new protos.google.showcase.v1beta1.CreateStreamingSequenceRequest()
+
+  let firstDelay = new protos.google.protobuf.Duration();
+  firstDelay.nanos=150;
+
+  let firstStatus = new protos.google.rpc.Status();
+  firstStatus.code=14;
```

Comment:
Let's use enum keys for codes, will make it a little bit more readable. Can be taken from https://github.com/googleapis/gax-nodejs/blob/main/src/status.ts. 14 is `Status.UNAVAILABLE`.